nisystemlink/utilities/data_exporter/_api_clients/_clients/_product_client.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+from nisystemlink.clients.core import HttpConfiguration
+from nisystemlink.clients.product import ProductClient as SystemLinkProductClient
+from nisystemlink.clients.product.models import (
+    Product,
+    ProductField,
+    QueryProductsRequest,
+)
+from nisystemlink.utilities.data_exporter._api_clients._constants._http_constants import (
+    HttpConstants,
+)
+
+
+class ProductClient:
+    __api_key: str | None
+    __systemlink_uri: str | None
+
+    __product_client: SystemLinkProductClient
+
+    def __init__(self, api_key: str | None, systemlink_uri: str | None) -> None:
```

Comment:
```suggestion
    def __init__(self, api_key: Optional[str] = None, systemlink_uri: Optional[str] = None) -> None:
```

apply the same suggestion for other clients